sass/_SearchInterface.scss

```diff
@@ -36,6 +36,10 @@
   * {
     @include box-sizing(border-box);
   }
+
+  .coveo-error {
```

Comment:
Could conflict with current implementations, but I would agree that there is a very, _very_ low probability of breaking anything :)